intelmq/bots/parsers/shadowserver/parser.py

```
@@ -40,6 +47,11 @@ class ShadowserverParserBot(ParserBot):
     overwrite = False
 
     def init(self):
+        config.set_logger(self.logger)
+        try:
+            config.update_schema()
+        except Exception as e:
+            self.logger.warning("Schema update failed: %s." % e)
```

Please do not do an unconditional update on every change. Please allow setting a parameter whether the bot should update the schema or not. This is unfortunately a debugging hell, and in addition hard to monitor – a warning in log will mostly be silently ignored, and we would not recognize it for the long time. I'd like to have a control over this behavior, e.g. disable automated update when I have a cronjob configured.

---

Now controlled by the `auto_update` parameter.